insights/parsers/libssh_config.py

```diff
@@ -0,0 +1,80 @@
+"""
+Libssh configuration files
+==========================
+
+This module provides the paths of configuration file for libssh package
+
+Parsers provided by this module are:
+
+LibsshClientConfig - file ``/etc/libssh/libssh_client.config``
+--------------------------------------------------------------
+
+LibsshServerConfig - file ``/etc/libssh/libssh_server.config``
+--------------------------------------------------------------
+"""
+
+import re
+from insights.specs import Specs
+from insights.core import CommandParser
+from .. import parser, get_active_lines
+from insights.parsers import SkipException
+
+# regex for matching the 'include' directives
+INCLUDE_RE = re.compile(r'^Include\s+\S+')
+
+
+class LibsshConfig(CommandParser, list):
+    """
+    Base class for libssh configuration file.
+
+    Sample output::
+
+        # Parse system-wide crypto configuration file
+        Include /etc/crypto-policies/back-ends/libssh.config
+        # Parse OpenSSH configuration file for consistency
+        Include /etc/ssh/sshd_config
+
+    Attributes:
+       self (list): List of included files
```

Comment:
`self` is not the `attribute` of this class. 

If the `include` directives/lines will be accessed often, better add a `obj.includes` 

And, It's not so reasonable to keep only the `include` directive of the `config` and discard all the others. It should be a common parser, and it's necessary to make it widely useful as much as it can, but not only keep the lines required by a single rule. You could refer to the existing parsers against other `config/conf` files and re-implement it.